app/policies/case/sar/internal_review_policy.rb

```diff
@@ -11,6 +11,10 @@ def can_respond?
     check_can_trigger_event(:respond)
   end
 
+  def edit?
+    super || user_is_an_approver_on_case?
+  end
```

Comment:
From the consistency between the permission policy and workflow, IMO, it would be better to work with the following. lines
  ```
     def edit?
        clear_failed_checks
        check_can_trigger_event(:edit_case)
     end
```
it should work same as above old code , and should work for manager and approver